SessionUtilitiesKit/Database/Models/KeyValueStore.swift

```diff
@@ -0,0 +1,301 @@
+// Copyright © 2024 Rangeproof Pty Ltd. All rights reserved.
+
+import Foundation
+import GRDB
+
+public struct KeyValueStore: Codable, Identifiable, FetchableRecord, PersistableRecord, TableRecord, ColumnExpressible {
```

Comment:
This looks like it behaves pretty much the same as the `Setting` table - just wondering whether we can reuse that instead of adding a new table which duplicates the behaviour?

You could look at renaming the current table if that is a concern (there is a `try db.rename(table:, to:)` function which should be able to do this as part of a migration - would just need to make sure all the old `Setting` usage moves across)

Reply:
That makes sense to me. I managed to rename the `setting` table to `keyValueStore` and also clean up the `Setting.swift`. Should be all good to go.